src/components/OrderDetails.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <ion-card>
+    <ion-item lines="none" >
+      <ion-label>
+        <h1>{{order.customerName}}</h1>
+        <p>{{order.orderId}}</p>
+      </ion-label>
+      <ion-note slot="end">{{ order.orderDate }}</ion-note>
```

Comment:
I think this should be user readable date. We should use filters here that shows date based upon the use timezone